src/Multipart/UploadState.php

```diff
@@ -44,21 +45,54 @@ public function getUploadId()
         return $this->uploadId;
     }
 
+    /**
+     * Get the part size.
+     *
+     * @return int
+     */
     public function getPartSize()
     {
         return $this->partSize;
     }
 
+    /**
+     * Set the part size.
+     *
+     * @param $partSize
+     */
     public function setPartSize($partSize)
     {
         $this->partSize = $partSize;
     }
 
+    /**
+     * Marks a part as being uploaded.
+     *
+     * @param int   $partNumber
+     * @param array $partData
+     */
     public function markPartAsUploaded($partNumber, array $partData = [])
     {
         $this->uploadedParts[$partNumber] = $partData;
     }
 
+    /**
+     * Returns whether a part has been uploaded.
+     *
+     * @param int $partNumber
+     *
+     * @return bool
+     */
+    public function hasPartBeenUploaded($partNumber)
```

Comment:
Should this be named `isUploaded` to be consistent with the other `is*` methods?